paddle/fluid/operators/batch_norm_mkldnn_op.cc

```diff
@@ -80,6 +80,7 @@ class BatchNormMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
     const float epsilon = ctx.Attr<float>("epsilon");
     const float momentum = ctx.Attr<float>("momentum");
     const bool is_test = ctx.Attr<bool>("is_test");
+    const bool fuse_with_relu = ctx.Attr<bool>("fuse_with_relu");
```

Comment:
why should we add `fuse_with_relu` attribute here? 
```
t = fluid.InferenceTranspiler()
t.transpile(infer_prog, place)
```
is enough. Do you mean there is `mkldnn::fuse_bn_relu` function in MKLDNN?

Reply:
`mkldnn::fuse_bn_relu` is a flag for MKLDNN batch norm telling it to execute relu along with batch norm.

Reply:
If we execute `fuse_batch_norm` always (no `not ise_mkldnn` if), then `fuse_relu` makes sense only in case where there is no conv before batch norm. I don't know if such case in fact ever exists.

If no, then I can skip this PR and create similiar one for the training, which I have already completed.

Reply:
> where there is no conv before batch norm

For DenseNet https://github.com/liuzhuang13/DenseNet, there is BN+Relu+Conv, thus, `fuse_relu` is useful in this case.

> I can skip this PR and create similiar one for the training

`fuse_relu` for training is needed as well.